tests/openvino/native/data/test_fq_remove_gt.json

```diff
@@ -0,0 +1,508 @@
+{
+    "FakeQuantize_5": {
```

Comment:
Do we drop all `FakeQuantize`'s from the model or just the some required cases to check?

Reply:
We take every FQ and search for additional FQs to remove. I agree that we can reduce the number of checks. But this test is fast enough. So I don't see reasons to do this.